addons/website_event_sale/views/event_templates.xml

```diff
@@ -9,7 +9,7 @@
 
 <template id="index" inherit_id="website_event.events_list" name="Event's Ticket">
     <xpath expr="//div[@t-foreach='event_ids']//footer" position="inside">
-        <t t-if="event.state in ['draft', 'confirm'] and event.event_ticket_ids">
+        <t t-if="(event.start_sale_date and event.end_sale_date and event.start_sale_date.date() &lt; datetime.date.today() &lt; event.end_sale_date.date()) and event.event_ticket_ids">
```

Comment:
Same here, you have code in second commit that is rewritten afterwards. either write the right one, either merge commit, either remove it, but we don't need unnecessary code in a feature task.